server/services/gameLogic.js

```diff
@@ -73,7 +73,7 @@ exports.quitGame = player => {
         socket.emitPlayers(getPlayers());
         return true;
     } else {
-        // Game has started, player can't quit
+        // Game has started, player can't quit, server is responsible for generating the error
         return false;
     }
```

Comment:
There are a few place where you're just returning in the `else` and you don't need the `else`.